tests/lib/rules/lighter-video.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const { RuleTester } = require("../../../lib/rule-tester");
+
+const rule = require( '../../../lib/rules/lighter-video' );
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2021,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
+  }
+});
+
+
+ruleTester.run('lighter-video', rule, {
+  valid: [
+  
+    {
+      code: "console.log('Hello, world!');",
```

Comment:
this test has nothing to do with this rule

Reply:
Okay, I've removed this test.